### PR TITLE
Arc and Azure CLI extensions GA + bug fix for SQL MIAA update

### PR DESCRIPTION
**Pull request by candiceye**
- Releasing the arc and azcli extensions out of preview and into GA.
- Updated the az command "edit" to be "update", in line with new Azure CLI arcdata extension commands
- Azure CLI arcdata SQL MIAA update now works for both direct and indirect modes (bug fix)
- Azure CLI arcdata SQL MIAA backup now works for both direct and indirect modes (bug fix)

### CODE REVIEW

**Review by alanrenmsft — APPROVED** [review]

File: extensions/arc/package.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
   "description": "%arc.description%",
   "version": "0.11.0",
   "publisher": "Microsoft",
-  "preview": true,
+  "preview": false,
```
Comment (Charles-Gagnon) [inline extensions/arc/package.json:7]: You can just remove this completely

File: extensions/azcli/package.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
   "description": "%azcli.arc.description%",
   "version": "0.3.0",
   "publisher": "Microsoft",
-  "preview": true,
+  "preview": false,
```
Comment (Charles-Gagnon) [inline extensions/azcli/package.json:7]: remove

File: extensions/arc/package.json [hunk 3]
```diff
@@ -4,7 +4,7 @@
   "description": "%arc.description%",
   "version": "0.11.0",
```
Comment (Charles-Gagnon) [inline extensions/arc/package.json:5]: Would suggest bumping this now so you don't have to do it later

File: extensions/azcli/package.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
   "description": "%azcli.arc.description%",
   "version": "0.3.0",
```
Comment (Charles-Gagnon) [inline extensions/azcli/package.json:5]: Would suggest bumping this now

**Review by Charles-Gagnon — APPROVED** [review]
🎉

**Pull request merged**